index.html

```diff
@@ -238,13 +238,18 @@ <h3>Introduction</h3>
 <section>
 <h3>Resources Included in the <a>PerformanceResourceTiming</a>
 Interface</h3>
-<p>All resources <a data-cite="FETCH#concept-fetch">fetched</a> by
-the current <a data-cite="HTML#browsing-context">browsing</a> or
-worker [[WORKERS]] context's MUST be included as
+<p>All resource <a data-cite="Fetch#concept-request">Request</a>s
+<a data-cite="FETCH#concept-fetch">fetched</a> by the current
```

Comment:
What is the "current browsing context"?  That's a longstanding issue here...  The PerformanceResourceTiming is tied to a global and hence settings object.  That's the settings object that should presumably be relevant here?

One option might be that given a fetch we get its [**client**](https://fetch.spec.whatwg.org/#concept-request-client), if not null get the PerformanceResourceTiming for that settings object's global (need to word this carefully around workers) and use that.  I _think_ that will do the right thing for iframes, because loads due to attr changes use the iframe's nodee document's window as the client, which is what we want here.

Doing this on the browsing context level, if actually implemented that way (the way the spec is trying to define it here), would be a security hole: a PerformanceResourceTiming from an earlier page in the history would be required to see loads from later pages in the history.   That's clearly not desirable.